actix-server/Cargo.toml

```diff
@@ -17,18 +17,17 @@ edition = "2018"
 name = "actix_server"
 path = "src/lib.rs"
 
-[features]
-default = []
-
 [dependencies]
 actix-rt = { version = "2.0.0", default-features = false }
 actix-service = "2.0.0-beta.5"
 actix-utils = "3.0.0-beta.4"
 
 futures-core = { version = "0.3.7", default-features = false, features = ["alloc"] }
+futures-util = { version = "0.3.7", default-features = false }
```

Comment:
if it's just for task mod could be futures-task

Reply:
futures-task does not have `AtomicWaker`. That said it's exported with futures-core just doc hidden. So only futures-core is enough here.